tests/e2e/csi_static_provisioning_basic.go

```diff
@@ -1210,4 +1247,161 @@ var _ = ginkgo.Describe("Basic Static Provisioning", func() {
 		}()
 	})
 
+	/*
+		Verify static provisioning - import VMDK.
+		Presetup: Create VMDK with valid storage policy.
+
+		Test Steps:
+		1. Create VMDK with valid storage policy.
+		2. Create Resource quota
+		3. Create CNS register volume with above created VMDK
+		4. verify PV, PVC got created , check the bidirectional reference.
+	*/
+	ginkgo.It("[csi-supervisor] Verify static provisioning - import VMDK", func() {
+
+		var err error
+		ctx, cancel := context.WithCancel(context.Background())
+		log := logger.GetLogger(ctx)
+		defer cancel()
+
+		dataStoreType, err := defaultDatastore.Type(ctx)
+		gomega.Expect(err).NotTo(gomega.HaveOccurred())
+		log.Infof("Datastore type: %s", dataStoreType)
+		if dataStoreType == "vsan" {
+
+			curtime := time.Now().Unix()
+			randomValue := rand.Int()
+			val := strconv.FormatInt(int64(randomValue), 10)
+			val = string(val[1:3])
+			curtimestring := strconv.FormatInt(curtime, 10)
+			pvcName := "cns-pvc-" + curtimestring + val
+			log.Infof("pvc name :%s", pvcName)
+			namespace = getNamespaceToRunTests(f)
+
+			restConfig, storageclass, _ := staticProvisioningPreSetUpUtilForVMDKTests(ctx)
+
+			defer func() {
+				log.Infof("Delete storage class")
+				err = client.StorageV1().StorageClasses().Delete(ctx, storageclass.Name, metav1.DeleteOptions{})
+				gomega.Expect(err).NotTo(gomega.HaveOccurred())
+			}()
+
+			vmdk := GetAndExpectStringEnvVar(envVmdkDiskURL)
+			framework.Logf("VMDK path : %s", vmdk)
+			ginkgo.By("Create CNS register volume with VMDK")
+			cnsRegisterVolume := getCNSRegisterVolumeSpec(ctx, namespace, "", vmdk, pvcName, v1.ReadWriteOnce)
+			err = createCNSRegisterVolume(ctx, restConfig, cnsRegisterVolume)
+			gomega.Expect(err).NotTo(gomega.HaveOccurred())
+			log.Infof("waiting for some time for FCD to register in CNS and for cnsRegisterVolume to get create")
+			//TODO add polling wait
+			time.Sleep(time.Duration(100) * time.Second)
```

Comment:
there is a variable for pandora sync time set for 90s

Reply:
done